specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2021-02-10/service.json

```diff
@@ -8458,10 +9401,24 @@
           "description": "The agent version.",
           "type": "string"
         },
+        "agentExpiryDate": {
+          "format": "date-time",
+          "description": "Agent expiry date.",
+          "type": "string"
+        },
         "isReplicationAgentUpdateRequired": {
           "description": "A value indicating whether replication agent update is required.",
           "type": "boolean"
         },
+        "agentCertificateExpiryDate": {
```

Comment:
mark as read only

Reply:
done

---
In reply to: [592113873](https://github.com/AsrOneSdk/azure-rest-api-specs/pull/16#discussion_r592113873) [](ancestors = 592113873)